cmd/earthly/buildkit.go

```diff
@@ -110,31 +114,6 @@ func (app *earthlyApp) getBuildkitClient(cliCtx *cli.Context, cloudClient *cloud
 	return buildkitd.NewClient(cliCtx.Context, app.console, app.buildkitdImage, app.containerName, app.installationName, app.containerFrontend, Version, app.buildkitdSettings)
 }
 
-func (app *earthlyApp) handleTLSCertificateSettings(context *cli.Context) {
```

Comment:
didn't see much point in keeping this function now that it's only passing values from the config to the buildkitsettings struct.